app/Http/Controllers/HomeController.php

```diff
@@ -48,6 +48,30 @@ public function getTimeLeftBoxBackground(float $days)
         }
     }
 
+    /**
+    * @description Set "hours", "days" or nothing behind the remaining time
+    *
+    * @param  float  $days
+    * @param  float  $hours
+    *
+    * @return string
+    */
+    public function getTimeLeftBoxUnit(float $days, float $hours)
+    {
+        if ($days < 1)
+        {
+            if ($hours < 1)
+            {
+                return;
```

Comment:
don't return void if your return docblock says string
can also return null and make return type !string
this code can be simplified